lib/portlayer/logging/logging.go

```diff
@@ -135,11 +135,29 @@ func Join(h interface{}) (interface{}, error) {
 		return nil, fmt.Errorf("Type assertion failed for %#+v", handle)
 	}
 
+	var logFilePath string
+
 	VMPathName := handle.Spec.VMPathName()
 	VMName := handle.Spec.Spec().Name
+	VMID := handle.Spec.ID()
+	dsName := handle.Spec.Datastore.Name()
+
+	//dsType, err := handle.Spec.Datastore.Type(context.Background())
```

Comment:
actually delete this code; do not leave it commented out. if it's checked in we can always get it out of the git history if necessary.